<ui version="4.0" >
 <class>keyer</class>
 <widget class="QMainWindow" name="keyer" >
  <property name="geometry" >
   <rect>
    <x>0</x>
    <y>0</y>
    <width>361</width>
    <height>611</height>
   </rect>
  </property>
  <property name="windowTitle" >
   <string>QtLog - cwKeyer</string>
  </property>
  <widget class="QWidget" name="centralwidget" >
   <layout class="QGridLayout" name="gridLayout_4" >
    <item row="0" column="0" colspan="4" >
     <widget class="QFrame" name="frame" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout" >
       <item row="0" column="0" >
        <widget class="QPushButton" name="ButtonF3" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F1</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="lineEditF1" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="ButtonF4" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F2</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="lineEditF2" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QPushButton" name="ButtonF5" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F3 </string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="lineEditF3" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QPushButton" name="ButtonF6" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F4</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="lineEditF4" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="1" column="0" colspan="4" >
     <widget class="QFrame" name="frame_2" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <property name="frameShadow" >
       <enum>QFrame::Plain</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_2" >
       <item row="0" column="0" >
        <widget class="QPushButton" name="ButtonF7" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>217</red>
               <green>211</green>
               <blue>181</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>217</red>
               <green>211</green>
               <blue>181</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>217</red>
               <green>211</green>
               <blue>181</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F5</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1" >
        <widget class="QLineEdit" name="lineEditF5" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="ButtonF8" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>217</red>
               <green>211</green>
               <blue>181</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>217</red>
               <green>211</green>
               <blue>181</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>217</red>
               <green>211</green>
               <blue>181</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F6</string>
         </property>
        </widget>
       </item>
       <item row="1" column="1" >
        <widget class="QLineEdit" name="lineEditF6" >
         <property name="text" >
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0" >
        <widget class="QPushButton" name="pushButton" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>217</red>
               <green>211</green>
               <blue>181</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>217</red>
               <green>211</green>
               <blue>181</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>217</red>
               <green>211</green>
               <blue>181</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F7</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLineEdit" name="lineEditF7" />
       </item>
       <item row="3" column="0" >
        <widget class="QPushButton" name="pushButton_2" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>217</red>
               <green>211</green>
               <blue>181</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>217</red>
               <green>211</green>
               <blue>181</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="Button" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>217</red>
               <green>211</green>
               <blue>181</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string> F8</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLineEdit" name="lineEditF8" />
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0" colspan="4" >
     <widget class="QFrame" name="frame_3" >
      <property name="frameShape" >
       <enum>QFrame::StyledPanel</enum>
      </property>
      <layout class="QGridLayout" name="gridLayout_3" >
       <item row="0" column="1" colspan="5" >
        <widget class="QLineEdit" name="lineEditF9" />
       </item>
       <item row="1" column="1" colspan="5" >
        <widget class="QLineEdit" name="lineEditF10" />
       </item>
       <item row="2" column="0" >
        <widget class="QLabel" name="label_4" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>10</red>
               <green>55</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>10</red>
               <green>55</green>
               <blue>255</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>146</red>
               <green>145</green>
               <blue>144</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text" >
          <string> *   AR</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1" >
        <widget class="QLabel" name="label_5" >
         <property name="text" >
          <string>  =  BT</string>
         </property>
        </widget>
       </item>
       <item row="2" column="2" >
        <widget class="QLabel" name="label_6" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>30</red>
               <green>193</green>
               <blue>1</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>30</red>
               <green>193</green>
               <blue>1</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>146</red>
               <green>145</green>
               <blue>144</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text" >
          <string>   &lt;  SK</string>
         </property>
        </widget>
       </item>
       <item row="2" column="3" colspan="2" >
        <widget class="QLabel" name="label_7" >
         <property name="text" >
          <string>    (   KN</string>
         </property>
        </widget>
       </item>
       <item row="3" column="0" >
        <widget class="QLabel" name="label_8" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>144</red>
               <green>144</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>144</red>
               <green>144</green>
               <blue>0</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>146</red>
               <green>145</green>
               <blue>144</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text" >
          <string> !   SN</string>
         </property>
        </widget>
       </item>
       <item row="3" column="1" >
        <widget class="QLabel" name="label_9" >
         <property name="text" >
          <string>  &amp;  AS</string>
         </property>
        </widget>
       </item>
       <item row="0" column="0" >
        <widget class="QPushButton" name="pushButton_3" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F9</string>
         </property>
        </widget>
       </item>
       <item row="1" column="0" >
        <widget class="QPushButton" name="pushButton_4" >
         <property name="focusPolicy" >
          <enum>Qt::NoFocus</enum>
         </property>
         <property name="text" >
          <string>F10</string>
         </property>
        </widget>
       </item>
       <item row="3" column="2" colspan="2" >
        <widget class="QLabel" name="label_10" >
         <property name="palette" >
          <palette>
           <active>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>156</red>
               <green>46</green>
               <blue>46</blue>
              </color>
             </brush>
            </colorrole>
           </active>
           <inactive>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>156</red>
               <green>46</green>
               <blue>46</blue>
              </color>
             </brush>
            </colorrole>
           </inactive>
           <disabled>
            <colorrole role="WindowText" >
             <brush brushstyle="SolidPattern" >
              <color alpha="255" >
               <red>146</red>
               <green>145</green>
               <blue>144</blue>
              </color>
             </brush>
            </colorrole>
           </disabled>
          </palette>
         </property>
         <property name="text" >
          <string>   >  BK</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="3" column="0" >
     <widget class="QLabel" name="label_11" >
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>Seconds</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="3" column="1" >
     <widget class="QSpinBox" name="repeater" >
      <property name="focusPolicy" >
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </item>
    <item row="3" column="2" colspan="2" >
     <widget class="QLabel" name="label_3" >
      <property name="text" >
       <string>Repeater text = F1</string>
      </property>
     </widget>
    </item>
    <item row="4" column="0" >
     <widget class="QLabel" name="label" >
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="text" >
       <string>WPM</string>
      </property>
      <property name="alignment" >
       <set>Qt::AlignCenter</set>
      </property>
     </widget>
    </item>
    <item row="4" column="1" >
     <widget class="QSpinBox" name="BoxSpeed" >
      <property name="focusPolicy" >
       <enum>Qt::NoFocus</enum>
      </property>
     </widget>
    </item>
    <item row="4" column="2" colspan="2" >
     <widget class="QLabel" name="label_2" >
      <property name="text" >
       <string>Page UP / Down</string>
      </property>
     </widget>
    </item>
    <item row="5" column="0" colspan="3" >
     <widget class="QPushButton" name="ButtonSave" >
      <property name="palette" >
       <palette>
        <active>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>204</red>
            <green>232</green>
            <blue>191</blue>
           </color>
          </brush>
         </colorrole>
        </active>
        <inactive>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>204</red>
            <green>232</green>
            <blue>191</blue>
           </color>
          </brush>
         </colorrole>
        </inactive>
        <disabled>
         <colorrole role="Button" >
          <brush brushstyle="SolidPattern" >
           <color alpha="255" >
            <red>204</red>
            <green>232</green>
            <blue>191</blue>
           </color>
          </brush>
         </colorrole>
        </disabled>
       </palette>
      </property>
      <property name="font" >
       <font>
        <weight>75</weight>
        <bold>true</bold>
       </font>
      </property>
      <property name="focusPolicy" >
       <enum>Qt::NoFocus</enum>
      </property>
      <property name="text" >
       <string>Save</string>
      </property>
     </widget>
    </item>
    <item row="5" column="3" >
     <widget class="QPushButton" name="ButtonExit" >
      <property name="text" >
       <string>Exit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar" >
   <property name="geometry" >
    <rect>
     <x>0</x>
     <y>0</y>
     <width>361</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuConfig" >
    <property name="title" >
     <string>  Config CWdaemon </string>
    </property>
    <addaction name="actionCwkeyer" />
   </widget>
   <addaction name="menuConfig" />
  </widget>
  <action name="actionCwkeyer" >
   <property name="text" >
    <string>daemon base config                    </string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
 <designerdata>
  <property name="gridDeltaX" >
   <number>4</number>
  </property>
  <property name="gridDeltaY" >
   <number>4</number>
  </property>
  <property name="gridSnapX" >
   <bool>true</bool>
  </property>
  <property name="gridSnapY" >
   <bool>true</bool>
  </property>
  <property name="gridVisible" >
   <bool>true</bool>
  </property>
 </designerdata>
</ui>
